EasyRestProjectNetTeam2/EasyRestPages/ManageWaitersPage.cs

```diff
@@ -1,10 +1,16 @@
-﻿using OpenQA.Selenium;
+﻿using EasyRestProjectNetTeam2.Decorator;
+using EasyRestProjectNetTeam2.EasyRestComponentsObj;
+using OpenQA.Selenium;
 using SeleniumExtras.PageObjects;
+using System.Collections.Generic;
 
 namespace EasyRestProjectNetTeam2.EasyRestPages
 {
     public class ManageWaitersPage : BasePage
     {
+
+        public AddEmployeeComponent AddEmploeeComponent { get; private set; }
```

Comment:
please rename to AddEmplo**y**eeComponent

Reply:
Done.